config/routes.rb

```diff
@@ -1,5 +1,11 @@
 Rails.application.routes.draw do
 
+  get 'company' => 'companies#show'
+
+  get 'company/edit' => 'companies#edit'
```

Comment:
I still prefer to use `resources :companies, only: [:show, :edit]`. However, that is not a critique, just my `0.02`.